app/assets/stylesheets/components/_costume-card.scss

```diff
@@ -1,41 +1,77 @@
 .costume-card {
-  overflow: hidden;
-  height: 200px;
-  background: rgba(255,255,255,0.5);
-  box-shadow: 0 1px 8px rgba(0,0,0,0.2);
   display: flex;
-  align-items: center;
-  margin-bottom: 8px;
-  color: rgba(5,5,5,0.7);
-  &:hover{
-    color: black;
-    box-shadow: 0 0 15px rgba(0,0,0,0.2);
-    background: white;
+  flex-direction: column;
```

Comment:
just curios about this flex-direction set to column - what does it do to the costume card?